Sources/FutureComposition.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  FutureComposition.swift
+//  Deferred
+//
+//  Created by Zachary Waldowski on 4/2/16.
+//  Copyright © 2014-2016 Big Nerd Ranch. Licensed under MIT.
+//
+
+extension FutureType {
+    /// Composes this future with another.
```

Comment:
This description doesn't really tell me much - "composes" is too vague to be useful. The `returns` doc OTOH is great. I'd recommend making that the main comment, dropping the `parameter other` bit (it doesn't tell you anything the types don't already), and leaving the `seealso`.